text/0018-rubygems-governance.md

```diff
@@ -0,0 +1,185 @@
+   - Additional supporting statements MAY be provided
+
+### 3.3 Voting Process
+1. After the notice period, a formal vote SHALL be held
+2. Voting SHALL remain open for 1 week
+3. Each owner gets one vote per candidate: Yes, No, or Abstain
+4. A candidate is accepted by **ordinary resolution** (simple majority of votes cast, excluding abstentions)
+5. Quorum MUST be met for the vote to be valid
+
+### 3.4 Implementation
+1. Accepted candidates SHALL be granted owner permissions within 1 week of vote completion
+2. New owners SHALL be provided with access to all relevant infrastructure and documentation
+
+## 4. Owner Removal Process
+
+### 4.1 Grounds for Removal
+Ownership removal MAY be iniated for:
+1. **Inactivity**: No meaningful project engagement for 12+ months
+2. **Policy Violation**: Serious violations of project policies or code of conduct
+3. **Breach of Trust**: Actions that undermine project governance or community trust
+
+### 4.2 Activity Requirements
+To maintain ownership, owners SHOULD demonstrate ongoing engagement through:
+- Code contributions or reviews
+- Participation in governance discussions
+- Infrastructure maintenance or support
+- Community engagement and support
+
+### 4.3 Removal Initiation
+1. Any owner MAY initiate a removal process by:
+   - Filing a formal proposal with specific grounds
+   - Providing evidence supporting the removal
+   - Notifying all owners with the proposal
+
+### 4.4 Notice and Discussion Period
+1. Removal proposals MUST be announced with at least 3 weeks notice
+2. During the notice period:
+   - The subject owner MUST be given opportunity to respond
+   - All owners MAY participate in discussion
+   - Additional evidence MAY be presented
+   - Mediation SHOULD be attempted when appropriate
+
+### 4.5 Voting Process
+1. After the notice period, a formal vote SHALL be held
+2. Voting SHALL remain open for 1 week
+3. Each owner (including the subject) gets one vote: Yes (remove), No (retain), or Abstain
+4. Removal requires **special resolution** (two-thirds supermajority of votes cast, excluding abstentions)
+5. Quorum MUST be met for the vote to be valid
+6. The subject owner MAY vote on their own removal
+
+### 4.6 Implementation
+1. If removal is approved, owner permissions SHALL be revoked within 48 hours
+2. The removed owner MAY retain maintainer access unless separately removed as maintainer
+3. Removed owners MAY be re-nominated through the standard addition process
+
+## 5. Emergency Procedures
+
+### 5.1 Emergency Removal
+In cases of immediate security threats, malicious activity, or legal violations:
+1. Any owner with technical capability MAY immediately suspend access
+2. Emergency action MUST be reported to all owners within 24 hours
+3. A formal removal vote MUST be initiated within 1 week
+4. If removal vote fails, access MUST be restored immediately
+
+### 5.2 Emergency Owner Addition
+In cases where owner count falls below operational minimums:
```

Comment:
What are the "operational minimums"?

Reply:
Yes, this is missing. I think 3 is the minimum, below which the policy can't work because you can't get a 2/3rds majority to remove someone with only 2 people.

Reply:
That information could probably just be added under _Definitions_, then capitalize the first letter + bold it like I suggested for definitions in general.